pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -331,7 +331,7 @@ def solve(self, *args, **kwds):
         extract_rc = ('rc' in model_suffixes)
 
         results = SolverResults()
-        results.problem.name = t1.name
+        results.problem.name = os.path.join(ws.working_directory, t1.name + '.gms')
```

Comment:
Should be fine, though hopefully nobody else was doing checks based on this value.